src/app/ubs-admin/components/ubs-admin-order-details-form/ubs-admin-order-details-form.component.html

```diff
@@ -0,0 +1,124 @@
+        <label class="col-12 col-sm-2 text-center form__label">Підтверджена к-ть</label>
+        <label class="col-12 col-sm-1 text-center form__label"></label>
+      </div>
+      <hr class="divider"/>
+      <div class="main">
+        <ul class="w-100 p-0">
+          <li class="d-flex justify-content-between align-items-baseline py-2">
+            <span class="col-12 col-sm-3">Безпечні відходи</span>
+            <span class="col-12 col-sm-2">120 л</span>
+            <span class="col-12 col-sm-2">250 грн</span>
+            <div class="col-12 col-sm-2 d-flex justify-content-center form-group count">
+              <input class="shadow-none form-control input p-2" type="number" min="0" max="999" placeholder="1" disabled/>
+            </div>
+            <div class="col-12 col-sm-2 d-flex justify-content-center form-group count">
+              <input class="shadow-none form-control input p-2" type="number" min="0" max="999" placeholder="" />
+            </div>
+            <div class="col-12 offset-sm-1 d-flex">
+            </div>
+          </li>
+          <li class="d-flex justify-content-between align-items-baseline py-2">
+            <span class="col-12 col-sm-3">Текстильні відходи</span>
+            <span class="col-12 col-sm-2">20 л</span>
+            <span class="col-12 col-sm-2">50 грн</span>
+            <div class="col-12 col-sm-2 d-flex justify-content-center form-group count">
+              <input class="shadow-none form-control input p-2" type="number" min="0" max="999" placeholder="1" disabled/>
+            </div>
+            <div class="col-12 col-sm-2 d-flex justify-content-center form-group count">
+              <input class="shadow-none form-control input p-2" type="number" min="0" max="999" placeholder="" />
+            </div>
+            <div class="col-12 offset-sm-1 d-flex">
+            </div>
+          </li>
+          <li class="d-flex justify-content-between align-items-baseline py-2">
+            <span class="col-12 col-sm-3">Текстильні відходи</span>
+            <span class="col-12 col-sm-2">120 л</span>
+            <span class="col-12 col-sm-2">300 грн</span>
+            <div class="col-12 col-sm-2 d-flex justify-content-center form-group count">
+              <input class="shadow-none form-control input p-2" type="number" min="0" max="999" placeholder="1" disabled/>
+            </div>
+            <div class="col-12 col-sm-2 d-flex justify-content-center form-group count">
+              <input class="shadow-none form-control input p-2" type="number" min="0" max="999" placeholder="" />
+            </div>
+            <div class="col-12 offset-sm-1 d-flex">
+            </div>
+           </li>
+          <li class="d-flex justify-content-between align-items-baseline py-2">
+            <span class="col-12 col-sm-3">Загальна кількість пакетів</span>
+            <span class="col-12 col-sm-2 offset-md-4 p-left">3</span>
+            <span class="col-12 col-sm-2 p-left">4</span>
```

Comment:
Why are you adding col-12 to all elements?
# With col-12 when screen < 480px
![image](https://user-images.githubusercontent.com/43093994/122367912-89793280-cf65-11eb-834c-6a778308de85.png)
# Without col-12 when screen < 480px
![image](https://user-images.githubusercontent.com/43093994/122368030-a7469780-cf65-11eb-93e6-c52a8d0e7dd6.png)

Reply:
fixed